src/prefect/utilities/dockerutils.py

```diff
@@ -127,15 +127,18 @@ class BuildError(Exception):
 def build_image(
     context: Path,
     dockerfile: str = "Dockerfile",
+    tag: Optional[str] = None,
     pull: bool = False,
     platform: str = None,
     stream_progress_to: Optional[TextIO] = None,
+    **kwargs,
```

Comment:
I'm not sure if there's a great way to doc this but it may be worth mentioning that any additional kwargs will get passed directly to the docker client's build command?

Reply:
Yeah, great idea.

Reply:
Added a docstring in [5dd3a6a](https://github.com/PrefectHQ/prefect/pull/10975/commits/5dd3a6a7b8497cc046bb56ab481133480ef33b95)!